cost-analyzer/templates/_helpers.tpl

```diff
@@ -1107,6 +1107,12 @@ Begin Kubecost 2.0 templates
       value: "true"
     - name: FEDERATED_CLUSTER # this ensures the ingester runs assuming federated primary paths in the bucket
       value: "true"
+    {{- if .Values.kubecostProductConfigs.standardDiscount }}
+    {{- if .Values.ingestionConfigmapName }}
+    - name: INGESTION_CONFIGMAP_NAME
+      value: {{ .Values.ingestionConfigmapName }}
+    {{- end }}
+    {{- end }}
```

Comment:
Currently this is defined within the the following conditionals:

```
{{- if eq (include "aggregator.deployMethod" .) "statefulset" }}
{{- if and (not .Values.kubecostModel.federatedStorageConfigSecret) (not .Values.kubecostModel.federatedStorageConfig) }}
```

Should we move INGESTION_CONFIGMAP_NAME out of these conditionals? Since it shouldn't depend on Aggregator being deployed as StatefulSet?

Also, needs `(.Values.kubecostProductConfigs).standardDiscount`

Reply:
Tbh the thought process was that this configuration essentially doesn't have a purpose outside of `(.Values.federatedETL).primaryCluster`, which forces `statefulset` IIRC; gating it behind that I thought was fine. There is no purpose in adding this to anything other than a fed etl architecture config.

Reply:
Makes sense, but what about future ingestionConfigs?

Reply:
I can imagine a (low-likelihood) case where we want to modify what was ingested but even in single cluster setup! Anyways, I think this is a minor point for now. And with respect to this PR it only defines `INGESTION_CONFIGMAP_NAME` which should only be set if the user provides an override. I'm good approving as is!

Reply:
If you want to add the following back to the top of your configmap, that may be good for the sake of consistency then.

```
{{- if eq (include "aggregator.deployMethod" .) "statefulset" }}
```